test/eslint.js

```diff
@@ -1,6 +1,12 @@
 var lint = require("mocha-eslint");
 
-var paths = ["*.js", "features/**/*.js", "plugins/**/*.js", "test/**/*.js"];
+var paths = [
+    "*.js",
+    "lib/**/*.js",
+    "features/**/*.js",
+    "plugins/**/*.js",
+    "test/**/*.js",
+];
```

Comment:
didn't even realize this file existed.  why do we have this?  we can use `npm run lint` to run lint on its own instead of running it with unit tests, which doesn't really make sense to me.

Reply:
It's just more convenient to test everything that we care about in one place instead of running unit tests and validate the code styling separately. Is there a case for running unit tests and not caring about linting or does it just not make sense because linting is not a unit test and you want to keep those concepts separate?

Reply:
The latter.  I was originally concerned that linting the code while istanbul was running would artificially inflate the coverage, but that turned out to be incorrect.

I think another case for separating the two is to add a separate GitHub action check for linting, so if one of the two checks fails it'll be immediately obvious what went wrong.

Reply:
Yeah that is a good point. I took out mocha-eslint and added a separate github lint action and an npm script that tests and lints so I can still run one command :)